### DISCUSSION TITLE
关于loginserver模板的问题

### DISCUSSION THREAD

**Discussion by donneyluckS** [cloudwu/skynet#1336] [General]
![1111](https://user-images.githubusercontent.com/60682638/105973255-7b666a80-60c7-11eb-8bc2-9598976c6013.png)

最近在学习skynet 
学习到了dh算法 了解到中间人攻击
请问这个登录流程中是怎么防范中间人攻击的呢？
我理解的好像并没有防范住
麻烦请大佬解答一下 不甚感谢

**Comment by cloudwu** [comment]
From https://github.com/cloudwu/skynet/issues/1335
